config/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+bases:
```

Comment:
Can you please add this at the top of the file :
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
```

Reply:
done